presto-main/src/main/java/io/prestosql/sql/planner/PartitioningScheme.java

```diff
@@ -91,6 +91,11 @@ public Partitioning getPartitioning()
         return outputLayout;
     }
 
+    /**
+     * Gets a column that is used in a task.
```

Comment:
Makes sense to make it clear that the column hash is being returned?
```suggestion
     * Gets the hash of the column used in a task.
```

Reply:
done